README.md

```diff
@@ -0,0 +1,118 @@
+# Django-import-export-extensions
+
+| Project   |     | Status |
+|-----------|:----|--------|
+| Versions  |     | [![PyPI - Python Versions](https://img.shields.io/pypi/pyversions/django-import-export-extensions)](https://pypi.org/project/django-import-export-extensions/)<br>[![PyPI - Django Versions](https://img.shields.io/pypi/frameworkversions/django/django-import-export-extensions)](https://pypi.org/project/django-import-export-extensions/) |
+| CI/CD     |     | [![Build status on Github](https://github.com/saritasa-nest/django-import-export-extensions/actions/workflows/checks.yml/badge.svg)](https://github.com/saritasa-nest/django-import-export-extensions/actions/workflows/checks.yml)<br>[![Test coverage](https://coveralls.io/repos/github/saritasa-nest/django-import-export-extensions/badge.svg?branch=main)](https://coveralls.io/github/saritasa-nest/django-import-export-extensions?branch=main)<br>[![Documentation Status](https://readthedocs.org/projects/django-import-export-extensions/badge/?version=latest)](https://django-import-export-extensions.readthedocs.io/en/latest/?badge=latest) |
+| Statistic |     | ![PyPI Downloads](https://static.pepy.tech/badge/django-import-export-extensions/month) |
+
+## Description
```

Comment:
What do you thinkl about adding table of contents https://github.com/saritasa-nest/saritasa-invocations#table-of-contents

Reply:
[The documentation](https://django-import-export-extensions.readthedocs.io/en/latest/index.html) already contains a table of contents, so I think it's better to keep `README.md` small